posts/2024-05-02-comparison-of-performance.ipynb

```diff
@@ -0,0 +1,521 @@
+{
```

Comment:
<blockquote>an order of magnitude <s>times</s> faster than our own list implementation.</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/coecms/coecms.github.io/pull/69/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>